api/_db.data-utils.js

```diff
@@ -79,7 +113,7 @@ const formFillTimeHeader = 'formFillTime';
 export const findById = (data, id) => {
 	assertValidId(id);
 
-	// Ignore human-readable headers(the first row), because those can change any time
+	// ignore human-readable headers(the first row), because those can change any time
```

Comment:
Why this change. It's not consistent in any way.

I usually have `// ` in the beginning of every comment (note the space) and capitalize the first char.

Just make them consistent at minimum.